source/PlayerInfo.cpp

```diff
@@ -1072,7 +1072,7 @@ map<const shared_ptr<Ship>, vector<string>> PlayerInfo::FlightCheck() const
 
 	auto flightChecks = map<const shared_ptr<Ship>, vector<string>>{};
 	for(const auto &ship : ships)
-		if(ship->GetSystem() && !ship->IsDisabled() && !ship->IsParked())
+		if(ship->GetSystem() && !ship->IsParked())
```

Comment:
If a ship is disabled and in the system, then it isn't landed. We should replace the check for whether the ship is disabled for whether the ship has a planet.
```suggestion
		if(ship->GetSystem() && ship->GetPlanet() && !ship->IsParked())
```